pkg/hsskey/hsskey.go

```diff
@@ -6,7 +6,8 @@
 // drives based on the following procedure:
 //  1. Via BMC, read a 32-byte secret seed known as the Host Secret Seed (HSS)
 //     using the OpenBMC IPMI blob transfer protocol
-//  2. Compute a password as follows:
+//  2. Via EEPROM, read a 32-byte HSS from EEPROM
```

Comment:
64? -- per `hostSecretSeedStructSize = 64` or I am misunderstanding ?

Reply:
updated name to secret seed. The seed is what used for unlocking disk